docs/source/release_notes/release-notes.rst

```diff
@@ -43,42 +43,45 @@ develop
          - Change
 
     *    - |userbreak| |changed|
-         - Projects ``atasdb-commons``, ``commons-annotations``, ``commons-api``, ``commons-executors``, ``commons-proxy``, and ``lock-api`` no longer force Java 6 compatibility. This eliminates the need for a Java 6 compiler to compile AtlasDB, but published artifacts for these projects will depend on Java 8 from now on.
+         - Projects ``atlasdb-commons``, ``commons-annotations``, ``commons-api``, ``commons-executors``, ``commons-proxy``, and ``lock-api`` no longer force Java 6 compatibility.
+           This eliminates the need for a Java 6 compiler to compile AtlasDB, but published artifacts for these projects will depend on Java 8 from now on.
```

Comment:
I've split the sentence at the comma, and replaced the second part with:

> However, users can no longer compile against AtlasDB artifacts using Java 6 or 7; they must use Java 8 if depending on these AtlasDB projects.

Not sure if we should say "depending on AtlasDB" since I take that to mean to be an AtlasDB client (i.e. using `atlasdb-config`, which already forces Java 8).